src/adv.py

```diff
@@ -33,10 +42,78 @@
 room['narrow'].n_to = room['treasure']
 room['treasure'].s_to = room['narrow']
 
+starting_knight = [Item("Steel Sword", "Something To Slice With."), Item("Lions Mane", "Increases Health Slightly")]
+starting_magi = [Item("Magic Spell Book", "Provides Knowledge Of Ancient Spells"), Item("Lions Mane", "Increases Health Slightly")]
+starting_assassin = [Item("Steel Dagger", "Something To Stab With."), Item("Lions Mane", "Increases Health Slightly")]
+
+
+
 #
 # Main
 #
+def main():
+    print(os.environ)
+    os.system('clear')
+    print("Welcome To Troy's Text Adventure Game\n")
+    print("1.) Start")
+    print("2.) Load")
+    print("3.) Exit")
+    option = input("-> ")
+    if option == "1":
+        start()
+    elif option == "2":
+        pass
+    elif option == "3":
+        sys.exit()
+    else:
+        print("Requires 1, 2, or 3 inputs")
+        main()
+
+
+def class_define(player_name):
+    print("Which Class Would You Like To Be {}: ".format(player_name))
+    print("1.) Knight")
+    print("2.) Magi")
+    print("3.) Assassin")
+
+    player_class = input('Class: \n').upper()
+    if player_class == "1" or player_class == "KNIGHT":
+        global starting_knight
+        default_inventory = starting_knight
+        class_name = "Knight"
+    elif player_class == "2" or player_class == "MAGI":
+        global starting_magi
+        default_inventory = starting_magi
+        class_name = "Magi"
+    elif player_class == "3" or player_class == "ASSASSIN":
+        global starting_assassin
+        default_inventory = starting_assassin
```

Comment:
I like that you start with different inventories for each different class. I guess that's the point, but still, good stuff.